server/controllers/applicant/getApplicantsStats.js

```diff
@@ -0,0 +1,26 @@
+const Applicant = require('../../database/models/applicant');
+
+const getApplicantsStats = (req, res, next) => {
+  Promise.all([
+    Applicant.find({ accepted: true }).countDocuments(),
+    Applicant.find({
+      applicationSubmittedDate: { $exists: true, $ne: null },
+    }).countDocuments(),
+    Applicant.find({
+      applicationSubmittedDate: null,
+    }).countDocuments(),
```

Comment:
we also need to count whole number of applications so you have to add the last line 
```suggestion
    Applicant.find({ accepted: true }).countDocuments(),
    Applicant.find({
      applicationSubmittedDate: { $exists: true, $ne: null },
    }).countDocuments(),
    Applicant.find({
      applicationSubmittedDate: null,
    }).countDocuments(),
    Applicant.find({}).countDocuments(),
```